docs/src/tutorial/receiving_messages.md

```diff
@@ -46,7 +50,7 @@ async fn connection_loop(stream: TcpStream) -> Result<()> {
             Some(idx) => (&line[..idx], line[idx + 1 ..].trim()),
         };
         let dest: Vec<String> = dest.split(',').map(|name| name.trim().to_string()).collect();
-        let msg: String = msg.trim().to_string();
+        let msg: String = msg.to_string();
```

Comment:
Why no trimming?

Reply:
Because `msg` is trimmed already 4 lines above. I would have preferred the `trim()` to happen here for clarity, but that's how it is presented in other samples of the code down the road (ie. no `trim()` in the following sections).